### ISSUE TITLE
question about running from source code

### ISSUE THREAD

**Issue by btwbtm** [hmorimitsu/ptlflow#20]
Hi, I want to make some modifications to this repo. However, after cloning the repo and running train.py, it says No module named 'pytorch_lightning'. I guess the command pip install dist/ptlflow-*.whl might help. But it download a torch with version>=1.7, which my GPU does not support. Is there any way that I can bypass this error?

**Comment by hmorimitsu** [comment]
Hi @btwbtm, if you want to modify the code, I think it is better to just install the requirements with pip, without building and installing ptlflow. I have updated the docs to mention this: https://ptlflow.readthedocs.io/en/latest/starting/installation.html

Regarding the torch version, you can just edit the file requirements.txt (if installing only requirements with pip), or the file setup.cfg (in case you want to build and install ptlflow) to decrease the version of torch (you will also have to decrease the version of torchvision).

However, I am not sure if everything will work with a lower version of torch. I think I tested a while ago, and some part of the code required something from pytorch 1.7+, but I don't remember what part was it.

**Comment by btwbtm** [comment]
Hi, thanks a lot for your prompt answer. It works now. Another small question is that, I see, flow papers typically train model on chairs, things, sintel, kitti in sequence. Does ptlflow support this training schedule?

**Comment by hmorimitsu** [comment]
Yes, you can do that. But, at least for now, you need to do one step at a time. For example, to replicate the training used in RAFT for Sintel, you could run 3 trainings in sequence as:

```bash
python train.py raft --train_dataset chairs
python train.py raft --train_dataset things --clear --resume_from_checkpoint /path/to/chairs/checkpoint
python train.py raft --train_dataset 100*sintel+200*kitti+5*hd1k+things --clear --resume_from_checkpoint /path/to/things/checkpoint
```

You can also provide more arguments to control the learning rate, number of steps, validation, etc.
